appcli/commands/service_cli.py

```diff
@@ -87,20 +99,20 @@ def start(ctx, force):
         def shutdown(ctx, force):
             self.__shutdown(ctx, force)
 
-        @click.command(
-            help="Stops the system (deprecated - use shutdown).", hidden=True
+        @service.command(
+            help="Stops all services. Alias to 'shutdown' command.", hidden=True
```

Comment:
It might be removed, or it might get reworked. I don't think its worth stating in the help text as it's impl detail which end users don't really need to worry about

Reply:
So the reason I explicitly stated it's an alias was to reduce end-user confusion about which of the two commands to use since the help text is otherwise identical.
I'll remove it anyway.